mkdocs.yml

```diff
@@ -20,6 +20,7 @@ plugins:
 markdown_extensions:
   - toc:
       permalink: true
+      toc_depth: 2
```

Comment:
I added this setting so that only headings up to level 2 are used to display the table of contents for a page in the right-hand menu.